### DISCUSSION TITLE
By querying to weather.gov/alerts receiving the list of response having Three Id's. how can we distinguish between them

### DISCUSSION THREAD

**Discussion by developerToday** [weather-gov/api#437] [Q&A]
I'm querying to wearher.gov endpoint: **https://api.weather.gov/alerts/active?point=34.04,-111.09** 

I see there are three urn id's in the response as below: 
<img width="1640" alt="Screen Shot 2021-07-23 at 2 09 26 PM" src="https://user-images.githubusercontent.com/68301254/126830204-2fbb1479-4daf-425e-b510-03fb83005eed.png">

Not sure why there are three id's in which two id's are inside the reference object. Can someone help me to understand what it is. if any documentation provided it will be useful.

**Comment by StephenClouse** [comment]
Everything from /alerts is based on the [OASIS Common Alerting Protocol](http://docs.oasis-open.org/emergency/cap/v1.2/CAP-v1.2-os.html).

References here indicate previous alert messages that this message supersedes.
